src/database/models/novels.js

```diff
@@ -0,0 +1,9 @@
+export default (sequelize, DataTypes) => {
```

Comment:
There's already a novel model and migration with the required columns and association, you should use that and not create a new one

Reply:
okay